src/main/java/com/msalvatore/graphqlplayground/entity/Vehicle.java

```diff
@@ -0,0 +1,76 @@
+package com.msalvatore.graphqlplayground.entity;
+
+import javax.persistence.*;
+import java.io.Serializable;
+import java.time.LocalDate;
+
+@Entity
+public class Vehicle implements Serializable {
```

Comment:
Pretty sure implementing `Serializable` isn't necessary these days, and I can scrap the `serialVersionUID`. Will update in the next PR.